extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -88,6 +88,8 @@ export const RENDERING_TYPE_HEADER = 'X-AmpAdRender';
 /** @type {string} */
 const TAG = 'AMP-A4A';
 
+const NO_CONTENT_RESPONSE = 'No content in response.';
```

Comment:
Might be better to have it just be 'NO-CONTENT-RESPONSE' or something to indicate that it's not for human consumption.